### PR TITLE
💄(testimonies) new content impacting view: scroll through testimonies

### PR DESCRIPTION
**Pull request by manuhabitela**
since we're not sure on the actually wanted layout in figma I just made the wrapping div of testimonies overflow, styled a bit the scrollbar, and made it so that mouse users can "scroll" easily.

i guess this is acceptable for now and we change when we get specific layout info?

https://github.com/numerique-gouv/lasuite-landingpage/assets/221253/c0c42683-77ff-4b3f-a219-7788c6b19e1a

### CODE REVIEW

**Comment by lebaudantoine** [comment]
What you propose is acceptable for a first iteration.
What do you think of using the props `slidesPerView` on `SwiperWrapper` component. The `SwiperWrapper` will always be displayed, and we would update its props value based on the current window width. On desktop, the props `slidesPerView` would be equal to 3, the dot will indicate that more slides are available. On mobile the props `slidesPerView` would be equal to 1.

My suggestion might be a bit "hacky" and requiring too much JS for smth that could be done in pure HTML and CSS as you propose. Let's discuss it

**Comment by lebaudantoine** [comment]
I discussed it with @johann, he would prefer the latter suggestion.

**Comment by manuhabitela** [comment]
Done!

And now with keyboard nav we can read all testimonies (was missing).

https://github.com/numerique-gouv/lasuite-landingpage/assets/221253/a4c809d0-eebd-43ba-b51d-60cac3d232f7

File: src/sections/Testimonies.tsx [hunk 1]
```diff
@@ -116,16 +151,17 @@ export const Testimonies = () => (
...
+    <div className="w-full">
       <SwiperWrapper
         slides={data.map((testimony) => (
           <Card {...testimony} key={testimony.key} />
         ))}
+        breakpoints={{
+          1024: {
+            slidesPerView: 3,
+            spaceBetween: 28,
+          },
```
Comment (manuhabitela) [inline src/sections/Testimonies.tsx:163]: Sadly we can't rely on our tailwind tokens with swiper here, must use pixel values.

**Comment by manuhabitela** [comment]
Just need to remove my superfluous yarn.lock change

File: src/sections/Testimonies.tsx [hunk 2]
```diff
@@ -20,7 +20,10 @@ interface CardProps {
 }
 
 const Card = ({ title, quote, img, entity }: CardProps) => (
-  <div className="flex flex-col bg-white p-7 text-left flex-1">
+  <div
+    tabIndex={0}
```
Comment (lebaudantoine) [inline src/sections/Testimonies.tsx:24]: nit: when navigating using tab, the pagination disappears.

**Review by lebaudantoine — APPROVED** [review]

File: src/sections/Testimonies.tsx (hunk 2 above)
Comment (manuhabitela) [reply src/sections/Testimonies.tsx:24]: Yeah it's that way for everytime we have this behavior. Could be improved for sure but the important thing is we can access everything.

**Pull request merged**